_extensions/seedcase-theme/theme.scss

```
@@ -1,12 +1,9 @@
 /*-- scss:defaults --*/
 
-@import url('https://fonts.googleapis.com/css2?family=Noto Sans');
-@import url('https://fonts.googleapis.com/css2?family=Fira+Code');
-@import url('https://fonts.googleapis.com/css2?family=Maven+Pro');
+@import url('https://fonts.bunny.net/css?family=poppins:400roboto-mono:400');
 
-$font-family-sans-serif: "Maven Pro", sans-serif;
-$font-family-monospace: "Fira Code", monospace;
-$headings-font-family: "Noto Sans", sans-serif;
+$font-family-sans-serif: "poppins", sans-serif;
```

```suggestion
$font-family-sans-serif: "Poppins", sans-serif;
```

I believe it is case-sensitive.

---

Yea, I guess it is ok! Just haven't seen it like that before.